views/PrestaShop/Admin/Common/Grid/grid_panel.html.twig

```diff
@@ -18,16 +18,20 @@
  *#}
 <div class="card js-grid-panel" id="{{ grid.id }}_grid_panel" data-hook-name="{{ grid.name|split(' ')|first }}">
   {% block grid_panel_header %}
-    <div class="card-header js-grid-header">
-      <h3 class="d-inline-block card-header-title">
-        {{ grid.name }} ({{ grid.data.records_total }})
-      </h3>
-      {% block grid_actions_block %}
-        <div class="d-inline-block float-right">
-          {{ include('@PrestaShop/Admin/Common/Grid/Blocks/grid_actions.html.twig', {'grid': grid}) }}
-        </div>
-      {% endblock %}
-    </div>
+    {% if grid.name is not null or grid.actions.grid|length > 0 %}
```

Comment:
I don't see why `grid.actions.grid` is needed, it's not used in this template :thinking: